include/mysql/service_my_snprintf.h

```diff
@@ -37,42 +37,51 @@
 
   @return a number of bytes written to a buffer *excluding* terminating '\0'
 
-  @post
+  @note
   The syntax of a format string is generally the same:
-  % <flag> <width> <precision> <length modifier> <format>
-  where everything but the format is optional.
+  %[<flag>][<length>][.<precision>][<size modifier>]<format>[<format extension>]
+  where everything but the <format> is optional.
 
-  Three one-character flags are recognized:
+  Two one-character <flags> are recognized:
     '0' has the standard zero-padding semantics;
     '-' is parsed, but silently ignored;
-    '`' (backtick) is only supported for strings (%s) and means that the
-        string will be quoted according to MySQL identifier quoting rules.
 
-  Both <width> and <precision> can be specified as numbers or '*'.
-  If an asterisk is used, an argument of type int is consumed.
+  Both <length> and <precision> are the same as in the standard.
+  They can be specified as integers, or as '*' to consume an int argument.
 
-  <length modifier> can be 'l', 'll', or 'z'.
+  <size modifier> can be 'l', 'll', or 'z'.
 
-  Supported formats are 's' (null pointer is accepted, printed as
-  "(null)"), 'b' (extension, see below), 'c', 'd', 'i', 'u', 'x', 'o',
-  'X', 'p' (works as 0x%x), 'f', 'g', 'M' (extension, see below),
-  'T' (extension, see below).
+  Supported <format>s are 's' (null pointer is accepted, printed as "(null)"),
+  'c', 'd', 'i', 'u', 'x', 'X', 'o', 'p' (works as "0x%x"), 'f', and 'g'.
 
-  Standard syntax for positional arguments $n is supported.
+  The '$n' syntax for positional arguments is supported.
 
-  Extensions:
+  Format extensions:
 
-  Flag '`' (backtick): see above.
+    Format 'sQ'
+      quotes the string with '`' (backtick)s similar to "`%s`",
+      but also "escapes" existing '`'s in the string to '``' as in SQL ''''
```

Comment:
Do we use any formatting in code documentation, or simply plain text?

Reply:
I don't think anything parses the documentation, so just plain text.

Reply:
wait 🧐 `/**`… `@param[out]`… This is Doxygen.